aimmo-game/simulation/pickups.py

```diff
@@ -44,6 +51,10 @@ def serialise(self):
     def _apply(self, avatar):
         avatar.health += self.health_restored
 
+        # Make sure the health is capped at 100.
```

Comment:
Isn't that max number bound to change with say, achievements / points...? Ie can we have this max number not hard coded? cc @Spycho

Reply:
This is a wider question about how we process turns. It is possible for an avatar (on 50 health) in a single turn to get both 100 points health pickup and -80 points damage pickup?
Is the result 70 points (50+100-80), or is the result 20 points (50+100, limited to 100, - 80)?

Reply:
@CelineBoudier: I wasn't aware that this is what we're planning to do, so I can remove this cap or we can change it when we have the extra functionality of achievements or points in our game.

@duncanrussell-ocado: So there's a few things that you're misunderstanding about the game logic I think. Damage pickups don't actually damage the avatar, they give them a damage boost if they want to attack others (this is a near refactoring goal for sure). Another point is, one cell cannot hold two pickups, so the scenario isn't possible.

Reply:
So as discussed with Chris, this is actually a wider topic we need to cover. I'll make an issue about this. For now, we want any kind of capping (whether it's 10 or 100, it makes no difference for now) functionality because limitless HP isn't something we generally want? :)